### PR TITLE
ADEN-7398 # Add Kargo bidder

### PR DESCRIPTION
**Pull request by jacekbj**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/kargo.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+				MOBILE_IN_CONTENT: {
+					sizes: [
+						[300, 250]
+					]
+				}
+			}
+		};
+
+	function isEnabled() {
+		return adContext.get('bidders.kargo') && (!babDetection.isBlocking() || adContext.get('opts.wadIL'));
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/kargo.js]: `return adContext.get('bidders.kargo') && !babDetection.isBlocking()` please. Kargo isn't prepared for recovery

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/kargo.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+	function prepareAdUnit(slotName, config) {
+		return {
+			code: slotName,
+			sizes: config.sizes,
+			bids: config.sizes.map(function () {
+				return {
+					bidder: bidderName,
+					params: {
+						placementId: '_cGWUgEUv0T'
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/kargo.js]: Should we put it in the slot definition?

**Review by fraszczakszymon — APPROVED** [review]

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/kargo.js [hunk 3]
```diff
@@ -19,7 +20,9 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.kargo', [
 		};
 
 	function isEnabled() {
-		return adContext.get('bidders.kargo') && !babDetection.isBlocking();
+		return adContext.get('bidders.kargo') &&
+			!babDetection.isBlocking() &&
+			!trackingOptIn.geoRequiresTrackingConsent();  // Kargo does not work in GDRP geo
```
Comment (dmnsobczak) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/kargo.js]: GDRP > GDPR :trollface:

**Review by dmnsobczak — APPROVED** [review]

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/kargo.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+					],
+					placementId: '_cGWUgEUv0T'
+				}
+			}
+		};
+
+	function isEnabled() {
+		return adContext.get('bidders.kargo') &&
+			!babDetection.isBlocking() &&
+			!trackingOptIn.geoRequiresTrackingConsent();  // Kargo does not work in GDPR geo
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/kargo.js]: Not sure if it's needed

**Review by fraszczakszymon — APPROVED** [review]

**Review by dmnsobczak — APPROVED** [review]

**Pull request merged**